Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.21 AS build
+
+# renovate: datasource=github-releases depName=codecov/uploader
+ARG UPLOADER_VERSION=v0.6.2
+
+WORKDIR /src
+COPY . .
+RUN go build -ldflags '-s -w -extldflags "-static"' -o plugin-codecov
+RUN if [ $(arch) = "aarch64" ] ; then curl -s https://github.com/codecov/uploader/releases/download/${UPLOADER_VERSION}/codecov-aarch64 -o codecov; fi
+RUN if [ $(arch) = "x86_64" ] ; then curl -s https://github.com/codecov/uploader/releases/download/${UPLOADER_VERSION}/codecov-alpine -o codecov; fi
+RUN chmod +x codecov
+
+FROM alpine:3.18
```

Comment:
Do we even need alpine or could it be a scratch image?

Reply:
Good point. I don't really see a need, maybe for the certificates? But then, let's just try without the alpine part.

Reply:
You can copy cert from alpine and then have a scratch as base image

Reply:
Why is the cert part needed? What happens without it?

Reply:
It depends what the go binary does ... if it makes web requests to https sites it will need this file